persistence/sql/sql_process_store.go

```diff
@@ -91,28 +92,372 @@ func (p sqlProcessStoreImpl) StopProcess(ctx context.Context, request persistenc
 	return resp, err
 }
 
+func (p sqlProcessStoreImpl) markPendingStateAsAborted(
```

Comment:
After rebase, you will see that @zklgame has created a new way to mark the states aborted in a batch query, which is more efficient than the way that I did